src/app/installers.tsx

```diff
@@ -21,32 +21,35 @@ export default function Installers({
 }: {
   release: GithubRelease | null;
 }) {
-  const [selectors] = useDeviceSelectors(window.navigator.userAgent);
   const recommended: RecommendedDownload[] = [];
-  const currentDevice: CurrentDevice = {
-    windows: selectors.isWindows,
-    mac: selectors.isMacOs,
-    android: selectors.isAndroid,
-    linux: selectors.osName.toLowerCase() == "linux", // https://github.com/duskload/react-device-detect/issues/200
-    ios: selectors.isIOS,
-    firefox: selectors.isFirefox,
-    chrome: selectors.isChrome,
-    edge: selectors.isEdge,
-    safari: selectors.isSafari,
-    desktop: selectors.isDesktop,
-    mobile: selectors.isMobile,
-  };
+  if (typeof window !== "undefined") {
```

Comment:
I'm getting a "runtime error" calling out this exact line, but I don't know how to disable SSR on 15...

Reply:
Line 1 says "use client", shouldn't that be sufficient?

Reply:
See the added line under this... :)
I thought so too, and I wish - but apparently, no. It's still "prerendered" unless I set `{ ssr: false }` upon import, which is now only supported from other client components...?!

Reply:
![image](https://github.com/user-attachments/assets/4e09b985-1b43-4d02-91e3-b0b20047dc54)